cnxepub/formatters.py

```diff
@@ -539,6 +539,7 @@ def _replace_exercises(elem):
       xmlns:mod="http://cnx.rice.edu/#moduleIds"
       xmlns:md="http://cnx.rice.edu/mdml"
       xmlns:c="http://cnx.rice.edu/cnxml"
+      lang="{{ metadata['language'] }}"
```

Comment:
could/should this be `xml:lang="..."`? I'm not sure when this becomes an issue but I'm guessing it will at least be a problem in the RNG validator.